templates/index.html

```diff
@@ -68,13 +68,13 @@ <h2>Prerequisites</h2>
                 <h2 class="display-4 fw-bold">What is facebooc</h2>
                 <div class="col-lg-6 mx-auto">
                     <p class="lead mb-4">
-                        Facebooc is a social platform designed by C language. Because of the basic of C language, the platform can respond much faster than other platform which verifies the application power of C language.
+                        Facebooc is a social platform designed with C language. Based on C, its response time is much less than others', which performs the flexibility of C.
```

Comment:
This is not "designed" with C language. This project is implemented in C language.